tools/cibadmin.c

```diff
@@ -892,6 +892,49 @@ node_path_xml(pcmk__output_t *out, va_list args)
     return pcmk_rc_ok;
 }
 
+PCMK__OUTPUT_ARGS("cibadmin-rendered-acls", "const char *")
+static int
+rendered_acls_default(pcmk__output_t *out, va_list args)
+{
+    const char *rendered = va_arg(args, const char *);
+
+    if (rendered == NULL) {
+        return pcmk_rc_no_output;
+    }
+    return out->info(out, "%s", rendered);
+}
+
+PCMK__OUTPUT_ARGS("cibadmin-rendered-acls", "const char *")
+static int
+rendered_acls_xml(pcmk__output_t *out, va_list args)
+{
+    const char *rendered = va_arg(args, const char *);
+
+    /* At time of writing, this is exactly xml_output_xml(). However, rendered
+     * is not a standard string serialization of an XML document if the ACL
+     * render mode is color or text. So using the output_xml() method does not
+     * seem correct. We want to explicitly create a CData node.
+     */
```

Comment:
I don't see the point in duplicating xml_output_xml.  It seems like your main concern here is just the name, but I think a comment would be fine to explain it.

Reply:
We've previously also talked about how `out->output_xml` should start taking an `xmlNode *` instead of a `const char *`.  I think that's a reasonable change for a future PR.  As I previously mentioned, I think the only reason it doesn't do that is because all the early callers already had a string and as more callers were added, we never revisited that decision.

So it would be worth thinking about how whatever we want to do in `rendered_acls_xml` would work with that.  The best part about all the formatted output code is it's all private API.

Reply:
Since it is private API like you mentioned, I don't think we need to worry about how `rendered_acls_xml` would work with it. We're gonna be continue outputting the XML as CData for the foreseeable future regardless, since that's what we do in other tools and it simplifies the API schemas.

But yeah, that could be a good idea in the future.